### PR TITLE
add ZB-TTS01 support

### PR DESCRIPTION
**Pull request by xsp1989**

### CODE REVIEW

File: docs/devices/ZB-TTS01.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+| Model | ZB-TTS01 |
+| Vendor  | [Easyiot](/supported-devices/#v=Easyiot)  |
+| Description | This is a text-to-speech controller that can convert GB2312 Chinese, letters, and numbers into Chinese speech. |
+| Exposes | send tts, last received status, linkquality |
+| Picture | ![ZB-TTS01](https://www.zigbee2mqtt.io/images/devices/ZB-TTS01.jpg) |
+
+
+<!-- Notes BEGIN: You can edit here. Add "## Notes" headline if not already present. -->
+## Notes
+## 1 Protocol Overview
```
Comment (Koenkk) [inline docs/devices/ZB-TTS01.md:27]: I think these details shouldn't be here, a z2m users does not have to deal with all of this since it is abstracted away.

**Comment by xsp1989** [comment]
I've deleted this content，Thanks

File: docs/devices/ZB-TTS01.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+| Vendor  | [Easyiot](/supported-devices/#v=Easyiot)  |
+| Description | This is a text-to-speech controller that can convert GB2312 Chinese, letters, and numbers into Chinese speech. |
+| Exposes | send tts, last received status, linkquality |
+| Picture | ![ZB-TTS01](https://www.zigbee2mqtt.io/images/devices/ZB-TTS01.jpg) |
+
+
+<!-- Notes BEGIN: You can edit here. Add "## Notes" headline if not already present. -->
+## Notes
+
+## Because this speaker can only convert Chinese, only the Chinese version of the protocol instructions is provided.
```
Comment (Koenkk) [inline docs/devices/ZB-TTS01.md:28]: The same goes fro this, why is the protocol necessary in the user docs? The users don't have to deal with this. I suggest removing it entirely.

File: docs/guide/usage/exposes.md [hunk 3]
```diff
@@ -9,7 +9,17 @@ There are two types of exposes:
 - Generic: types like `numeric` and `binary`
 - Specific: represents a specific capability of a device like a `light` or `switch`.
 
-Both types will always have a `type` property. The generic types (except composite) will always have an `access` property and an optional `description` property. All generic types will always have a `name` property indicating the context and `label` property containing the name of the capability in the correct case and without using the underscore separator (e.g. `Device temperature`, `VOC`, `Power outage count`). All generic types will always have a `property` type indicating where the value is exposed on, usually this is equal to the name but in case when the `endpoint` is defined it is `name_endpoint`. The specific and the generic composite type will always have a `features` property, this is an array containing the generic exposes types. Optionally both types can have a property `endpoint`, indicating the device exposes this capability on a specific endpoint.
+Both types will always have a `type` property.
```
Comment (Koenkk) [inline docs/guide/usage/exposes.md:12]: Why was this changed? If it's not intentional, can you revert the changes to `exposes.md`?

**Comment by xsp1989** [comment]
This commit is not from my changes, maybe it's because I made a commit on the develop branch that caused this problem?

image

**Comment by Koenkk** [comment]
Thanks!
